src/test/a11y/amendAccessibilityReport.js

```diff
@@ -0,0 +1,49 @@
+const fs = require('fs');
+const path = require('path');
+
+
+function amendAccessibilityReport() {
+    amendHeading( /manage-user-a11y-audit\.html$/, 'AXE Accessibility Results - Manage User Page');
+    amendHeading(/search-user-a11y-audit\.html$/, 'AXE Accessibility Results - Search User Page');
+    amendHeading(/edit-user-a11y-audit\.html$/, 'AXE Accessibility Results - Edit User Page');
+    amendHeading(/generate-user-report-a11y-audit\.html$/, 'AXE Accessibility Results - Generate User Report Page');
+    amendHeading(/add-new-user-a11y-audit\.html$/, 'AXE Accessibility Results - Add New User Page');
```

Comment:
bit hardcoded - might be better to just put the file name in the header (maybe without the "html")

Reply:
Done